MyModTest/BaseTest.cs

```diff
@@ -1298,8 +1305,16 @@ protected virtual IEnumerator MakeDecisions(IDecision decision)
 
                     if (_numberOfChoicesInNextDecision != null)
                     {
-                        Assert.AreEqual(_numberOfChoicesInNextDecision, selectTurnTaker.Choices.Count(), "SelectTurnTakerDecision has the wrong number of choices.");
-                        _numberOfChoicesInNextDecision = null;
+                        var check = true;
```

Comment:
There doesn't seem to be a need for this "check" variable, this can be a single if statement. Same thing for the next 2 chunks.

Reply:
Went ahead and fixed it in the four places I found that block. :)